crates/gitbutler-core/src/ops/reflog.rs

```diff
@@ -88,11 +102,13 @@ fn set_oplog_ref(file_path: &PathBuf, sha: &str) -> Result<()> {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use gix::Repository;
+    use std::str::FromStr;
     use tempfile::tempdir;
 
     #[test]
-    fn test_set_target_ref() {
-        let (dir, commit_id) = setup_repo();
+    fn test_set_target_ref() -> anyhow::Result<()> {
```

Comment:
Even though I typically start with `.unwrap()` while it's unclear if things work or not, I finalize tests by moving them to `?` from `.unwrap()` so they are more readable. Most tests are 'read-only` after writing them, so it's helpful to leave them in the most readable state.